src/figuresClasses.ts

```diff
@@ -1,19 +1,93 @@
-export interface Figure {
+import {
+  INVALID_SIDE_LENGTH_ERROR,
+  GEOMETRIC_INCONSISTENCY_ERROR,
+  INCORRECT_RADIUS,
+  INVALID_RECTANGLE_SIDE_LENGTH,
+} from './utils/errorMessage';
 
-}
+type Shapes = 'triangle' | 'circle' | 'rectangle';
+type Colors = 'red' | 'green' | 'blue';
```

Comment:
But I'd better use `enum` here
In this case you will be able to do -> `public shape: Shapes = Shapes.triangle;` instead of `public shape: Shapes = 'triangle';`
And rename it:
`Shapes -> Shape`
`Colors -> Color`